src/kernel/caches.h

```diff
@@ -18,10 +23,10 @@ struct CacheSizes {
 
     CacheSizes(int64_t total_cache)
     {
-        block_tree_db = std::min(total_cache / 8, nMaxBlockDBCache << 20);
+        block_tree_db = std::min(total_cache / 8, MAX_BLOCK_DB_CACHE << 20);
         total_cache -= block_tree_db;
         // use 25%-50% of the remainder for disk cache, capped by the maximum.
-        coins_db = std::min({total_cache / 2, (total_cache / 4) + (1 << 23), nMaxCoinsDBCache << 20});
+        coins_db = std::min({total_cache / 2, (total_cache / 4) + (1 << 23), MAX_COINS_DB_CACHE << 20});
```

Comment:
re: In commit "kernel: Move kernel-related cache constants to kernel cache" (8e929e343e14b0228f6f4841ecf2d85a73afda3b)

Precedes this PR, but this code does not seem to make any sense. (1 << 23) is 8MiB which is the same as MAX_COINS_DB_CACHE. So this is effectively min(total / 2, total / 4 + 8MiB, 8Mib) with the middle argument not doing anything. Could be simplified to `std::min(total_cache / 2, MAX_COINS_DB_CACHE << 20)` with no change in behavior AFAICT.

Reply:
Heh, I guess I should have actually done the math here. Will remove.